applications/sink/zeromq-sink/README.adoc

```diff
@@ -0,0 +1,41 @@
+//tag::ref-doc[]
+= ZeroMQ Sink
+
+The "zeromq" sink enables sending messages to a ZeroMQ socket.
+
+== Input
+
+* `byte[]`
+
+== Output
+
+=== Payload
+
+N/A
+
+== Options
+
+The **$$zeromq$$** $$sink$$ has the following options:
+
+//tag::configuration-properties[]
+//end::configuration-properties[]
```

Comment:
Why is this section empty?
How have you generated them before for the `source` app?

Reply:
Not sure. Originally, I copied in one of the other files and modified it for this project. I also checked the plugin configuration and it seems to be correct unless I'm missing something in its configuration.

Reply:
Ok. Did you try `install` ? 
@sobychacko , what Maven command should we use to make props available in readers like this?
Thanks